build-system/tasks/check-sourcemaps.js

```diff
@@ -144,6 +146,14 @@ function checkSourcemapMappings(sourcemapJson, map) {
     'If this change is intentional, update the mapping related constants in ' +
     cyan('build-system/tasks/check-sourcemaps.js') +
     '.';
+
+  const expectedFirstLineFile = map.includes('mjs')
+    ? expectedFirstLineFileMjs
+    : expectedFirstLineFileJs;
+  const expectedFirstLineCode = map.includes('mjs')
+    ? expectedFirstLineCodeMjs
+    : expectedFirstLineCodeJs;
```

Comment:
Entirely style preference, but I feel like it might be clearer here to move all the strings in directly a la
```javascript
const expectedFirstLines = map.includes('.mjs')
  ? { file: 'src/core/mode/version.js', code: 'function version() {' }
  : { file: 'src/core/mode/prod.js', code: 'export function isProd() {' };
  ```

Reply:
As written, this is six identifiers that look very similar until you look closely

Reply:
Done